src/engine/controls/cuecontrol.cpp

```diff
@@ -607,8 +611,8 @@ void CueControl::seekOnLoad(mixxx::audio::FramePos seekOnLoadPosition) {
 }
 
 void CueControl::cueUpdated() {
-    //auto lock = lockMutex(&m_mutex);
-    // We should get a trackCuesUpdated call anyway, so do nothing.
+    // auto lock = lockMutex(&m_mutex);
+    //  We should get a trackCuesUpdated call anyway, so do nothing.
```

Comment:
Unrelated change? Befor clang format we had the rule to use "//" without space for commented code and "// " for information.

Reply:
this happens automatically with the pre-commit since the update about 10 days ago (was an update of some minutes)